### PR TITLE
Use all certificates in the chain when creating the ca-file for server-side OpenVPN configuration

### PR DESCRIPTION
**Pull request by scipiojr**
see attached comment.

### CODE REVIEW

**Comment by pfSense-git-updates** [comment]
Before this pull request can be accepted you must first sign a CLA as described at https://www.pfsense.org/about-pfsense/#cla. Please read for more details.

File: src/etc/inc/openvpn.inc [hunk 1]
```diff
@@ -1007,8 +1007,16 @@ function openvpn_reconfigure($mode, $settings) {
 		case 'server_tls':
 		case 'server_tls_user':
 		case 'server_user':
-			$ca = lookup_ca($settings['caref']);
-			openvpn_add_keyfile($ca['crt'], $conf, $mode_id, "ca");
+			// ca_chain() expects parameter to be passed  by reference. 
+			// avoid passing the whole settinsg array, as param names or
```
Comment (phil-davis) [inline src/etc/inc/openvpn.inc]: s/settinsg/settings/
And the line above has 2 spaces between "passed" and "by".

**Comment by scipiojr** [comment]
fixed the typos and signed cla by creating a pfsense account with icla subscription

**Comment by cbuechler** [comment]
merged to master for 2.4, thanks @scipiojr

**Pull request merged**